components/fetchers/group.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Buddypress\CLI\Command\Fetchers;
+
+use WP_CLI\Fetchers\Base;
```

Comment:
This is really good and could be used throughout our CLI commands. We already have home-grown versions, but we could migrate over to this syntax.

Reply:
Was thinking the same thing. o/